modules/ROOT/pages/kafka/kafka-connector-reference.adoc

```diff
@@ -5,252 +5,639 @@ endif::[]
 
 Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
 
-Kafka Connector v3.0
+Kafka Connector v4.0
 
 Anypoint Connector for Apache Kafka (Kafka Connector) allows you to interact with the Apache Kafka messaging system, and enable seamless integration between your Mule app and an Apache Kafka cluster, using Mule runtime.
 
-Release Notes: xref:release-notes::connector/kafka-connector-release-notes-mule-4.adoc[Kafka Connector Release Notes]
+Release Notes: xref:release-notes::connector/kafka-connector-release-notes-mule-4.adoc[Kafka Connec
+
 
 == Configurations
 ---
-[[kafka-consumer-config]]
+[[consumer-config]]
 === Consumer Configuration
 
+
 ==== Parameters
 [%header,cols="20s,20a,35a,20a,5a"]
 |===
 | Name | Type | Description | Default Value | Required
 |Name | String | The name for this configuration. Connectors reference the configuration with this name. | | x
-| Connection a| * <<kafka-consumer-config_basic-kafka-consumer-connection, Kafka Basic Consumer Connection>> 
-* <<kafka-consumer-config_kerberos-kafka-consumer-connection, Kafka Kerberos Consumer Connection>> 
-* <<kafka-consumer-config_kerberos-ssl-kafka-consumer-connection, Kafka Kerberos SSL Consumer Connection>> 
-* <<kafka-consumer-config_ssl-kafka-consumer-connection, Kafka SSL Consumer Connection>> 
+| Connection a| <<consumer-config_consumer-plaintext-connection, Consumer Plaintext Connection>>
  | The connection types to provide to this configuration. | | x
-| Expiration Policy a| <<ExpirationPolicy>> |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before the runtime considers it eligible for expiration. This does not mean that the platform expires the instance at the exact moment that it becomes eligible. Mule purges the instances as appropriate. |  | 
+| Default acknowledgment mode a| Enumeration, one of:
+
+** AUTO
+** MANUAL
+** IMMEDIATE
+** DUPS_OK a|  Defines the way that the Kafka broker instance is notified of the consumption of messages. 
+
+* AUTO: Messages are committed only if the flow finishes successfully. 
+* MANUAL: The user must commit manually through the Commit operation. 
+* IMMEDIATE: Mule automatically commits the messages upon reception and before triggering the flow. 
+* DUPS_OK: Same as the MANUAL mode, but the commit is made asynchronously, which can lead to duplicate records. | `AUTO` | 
+| Default listener poll timeout a| Number |  The time, in time units, spent waiting to do a poll if data is not available in the buffer (fetched). If no value is set, returns immediately with any records that are available currently in the buffer, else returns empty. Must not be negative. |  `100` | 
```

Comment:
Need a subject for the second sentence. "If no value is set, the connection request??? returns immediately..."